apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -1699,6 +1699,231 @@ properties:
     osver: {ios: {min: "9.1"}}
     since: "5.1.0"
 
+  - name: CONTENT_TYPE_NAME
+    summary: Tell keyboard that content type is, name.
```

Comment:
That sounds weird. Please use the [Apple phrasing](https://developer.apple.com/documentation/uikit/uitextcontenttype) for all constants.